Crashlytics/Crashlytics/Handlers/FIRCLSMachException.c

```diff
@@ -108,11 +108,13 @@ static bool FIRCLSMachExceptionThreadStart(FIRCLSMachExceptionReadContext* conte
 
   if (pthread_attr_init(&attr) != 0) {
     FIRCLSSDKLog("pthread_attr_init %s\n", strerror(errno));
+    pthread_attr_destroy(&attr);
```

Comment:
I am checking the [manual](https://linux.die.net/man/3/pthread_attr_destroy) for pthread_attr_destroy checking nullable for `attr` might be a safer implementation.
```
 if (&attr != NULL) {
        s = pthread_attr_destroy(& attr);
 }
```

Reply:
Looks fine since address will always none null
<img width="634" alt="Screenshot 2024-07-16 at 11 54 51 AM" src="https://github.com/user-attachments/assets/2bb869d9-6efe-4922-85a7-d7e68d67b679">